src/PlusDataCollection/PlusSerialLine.cxx

```diff
@@ -235,3 +235,47 @@ unsigned int SerialLine::GetNumberOfBytesAvailableForReading() const
   return 0;
 #endif
 }
+
+PlusStatus SerialLine::SetDTR(bool onOff)
+{
+#ifdef _WIN32
+  DWORD dwFunc = CLRDTR;
+  if (onOff)
+  {
+    dwFunc = SETDTR;
+  }
+  if (EscapeCommFunction(CommHandle, dwFunc))
+  {
+    return PLUS_SUCCESS;
+  }
+  else
+  {
+    return PLUS_FAIL;
+  }
+#else
+  LOG_ERROR("SerialLine::SetDTR() is only implemented on Windows");

```

Comment:
Linux implementation does not seem complicated. Could you add it?

See http://xanthium.in/Controlling-RTS-and-DTR-pins-SerialPort-in-Linux
and https://github.com/PlusToolkit/ndicapi/blob/master/ndicapi_serial_unix.cxx#L162

Not sure about Apple, but it would be great to include all OS's.
In case it helps: https://github.com/PlusToolkit/ndicapi/blob/master/ndicapi_serial_apple.cxx